.github/workflows/ci.yml

```diff
@@ -70,8 +68,18 @@ jobs:
                         </settings>' > ~/.m2/settings.xml
             - name: print file
               run: cat ~/.m2/settings.xml
-            - name: Maven Build
-              run: mvn clean install 
+            # Grant execute permission to the gradlew script
+            - name: Grant execute permission for gradlew
+              run: chmod +x gradlew
+
+            # Build the project with Gradle
+            - name: Build with Gradle
+              env:
+                packageUser: ${{ secrets.BALLERINA_BOT_USERNAME }}
+                packagePAT: ${{ secrets.BALLERINA_BOT_TOKEN }}
+                JAVA_OPTS: -DBALLERINA_DEV_COMPILE_BALLERINA_ORG=true
+              run: |
+                ./gradlew build
             - name: Ballerina Download
```

Comment:
lets use `setup-ballerina` action instead of downloading a predefined ballerina version manually

Reply:
apply where necessary in other workflow files as well